Wireframe/css/style.css

```diff
@@ -0,0 +1,131 @@
+@media (prefers-reduced-motion:no-preference){
```

Comment:
I thought wireframe and form-controls were supposed to be two different PRs?  Never mind, I'll review both here.

Reply:
I made two different PRs for each one. doesn't it show? could you please let me know how to check for it?